packages/cardano-services/src/Program/options/common.ts

```diff
@@ -41,27 +42,23 @@ export type CommonProgramOptions = {
 
 export const withCommonOptions = (command: Command, apiUrl: URL) => {
   addOptions(command, [
-    newOption('--api-url <apiUrl>', Descriptions.ApiUrl, 'API_URL', (url) => new URL(url), apiUrl),
+    newOption('--api-url <apiUrl>', Descriptions.ApiUrl, 'API_URL', urlValidator(Descriptions.ApiUrl), apiUrl),
     newOption('--build-info <buildInfo>', Descriptions.BuildInfo, 'BUILD_INFO', buildInfoValidator),
+    newOption('--dump-only <true/false>', Descriptions.DumpOnly, 'DUMP_ONLY', (dumpOnly) =>
```

Comment:
I commonly saw this kind of option named `--dry-run`. We could use the same if you agree that users would intuitively use `--dry-run` for this kind of functionality. 
It is just a suggestion.

Reply:
That option name is already used by the projector for a different purpose

Reply:
To me it looks kinda similar. Perhaps we could consolidate the 2 options? :thinking: 
```
  DryRun = 'Initialize the projection, but do not start it',
```

Reply:
It's not so easy.
The newly added `--dump-only` option, dumps the options and exits before actually starting the program while the `--dry-run` option, for the projector, let it to perform the initialization, and for the blockfrost worker actually starts everything but don't run the refresh job.
We use the `--dry-run` options in some tests to check the initialization: we should also rework these tests.

Reply:
I understand it's not so easy, but now we have 2 slightly different options for what it seems to be the same purpose. Perhaps we should create a separate ticket to consolidate the options?